utils/chains.json

```diff
@@ -77,5 +77,12 @@
         "decimals": 18
       }
     }
+  },
```

Comment:
to add a new chain, you need to also add all inside `chain_swap`

Example below:

```
    "chain_swap": {
      "chainId": "0xA4B1",
      "blockExplorerUrls": ["https://arbiscan.io"],
      "chainName": "Arbitrum One",
      "rpcUrls": ["https://arb1.arbitrum.io/rpc"],
      "nativeCurrency": {
        "name": "AETH",
        "symbol": "AETH",
        "decimals": 18
      }
     

```

Reply:
added!